conda/testing/notices/fixtures.py

```diff
@@ -35,6 +36,7 @@ def notices_mock_fetch_get_session():
 @pytest.fixture(scope="function")
 def conda_notices_args_n_parser():
     parser = generate_parser()
-    args = parser.parse_args(["notices"])
+    args = parser.parse_args(["notices", "-c", DEFAULTS_CHANNEL_NAME])
```

Comment:
Using the full option argument makes everything less magical and easier to read:

```suggestion
    args = parser.parse_args(["notices", f"--channel={DEFAULTS_CHANNEL_NAME}"])
```